lib/uplinkc/access_scope.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+// #include "uplink_definitions.h"
+import "C"
+
+import (
+	"fmt"
+
+	libuplink "storj.io/storj/lib/uplink"
+)
+
+//export parse_scope
+// parse_scope parses an Scope
+func parse_scope(val *C.char, cerr **C.char) C.ScopeRef {
+	scope, err := libuplink.ParseScope(C.GoString(val))
+	if err != nil {
+		*cerr = C.CString(fmt.Sprintf("%+v", err))
+		return C.ScopeRef{}
+	}
+
+	return C.ScopeRef{universe.Add(scope)}
```

Comment:
```suggestion
	return C.ScopeRef{_handle: universe.Add(scope)}
```

nit: For consistency with other parts of this package.

Reply:
changed :+1: